test/testdata/Iat-invalidAddenda13.ach

```diff
@@ -4,7 +4,7 @@
 710ANN000000000000100000928383-23938          BEK Enterprises                          0000001
 711BEK Solutions                      15 West Place Street                             0000001
 712JacobsTown*PA\                     US*19305\                                        0000001
-713Wells`Fargo                        01231380104                         US           0000001
+713WellsFargo                        01231380104                         US           0000001
```

Comment:
This is a bit tricky, but we should avoid using the `DEL` character as an invalid char in this testdata. Is there another invalid character that would be more explicit?

Reply:
Any of the 0x00 -> 0x1F control characters would suffice, too.

Unfortunately with this change, we have no more invalid printable ASCII characters.

Reply:
One other option would be to use an "extended ASCII" character and re-encode the test files as ISO-8559 - we could then use any of the characters in e.g. the ISO 8859-1 code page, which Go will still parse as a single byte rune.

Reply:
Could we try the bell character? `BEL = 0x07 // '\a' Bell`
It should show up in Github as
```bash

```

Reply:
If you'd like, sure. However, it'll make your shell bleep at you when you look at `git log -p` :stuck_out_tongue:

Reply:
If we can let's avoid bell/alarm characters. Those have caused annoyances in the past on other projects. Side effects such as triggering hardware sounds isn't ideal.

Reply:
I made this PR: https://github.com/moov-io/ach/pull/827

If that works, we should be able to use any invalid character from utf-8 like `日`

Reply:
Yeah, that's simpler and won't trip up editors. As long as we're good with the files being UTF-8, happy to throw a multi-byte character in instead of an ASCII control character.

Reply:
I tested it locally, but unfortunately, `Parse()` still parses the record as bytes. So using an invalid character UTF-8 character in tests won't work for now.
Link to the Parse() method: https://github.com/moov-io/ach/blob/080e8fcf72fda916f380335bc5b38b63259ef4d7/fileHeader.go#L123

Instead of the `bell` to avoid the sound, could we try `ESC = 0x1B // '\e' Escape`? I tested it locally, and it works with the test:

![image](https://user-images.githubusercontent.com/29054071/104074196-52d40a80-51c4-11eb-950c-0741e87bb928.png)

@Deiz thanks for being patient while we try out these weird edge cases :)

Reply:
ESC may still trigger terminal misbehaviour when printed, so I went with SUB (the replacement character, �) which a) is printable, b) is disallowed by the spec because it's below 0x1F in ASCII.